bedrock/firefox/templates/firefox/dashboard-base.html

```diff
@@ -7,7 +7,7 @@
 {% block page_title_prefix %}{% endblock %}
 {% block page_title %}{% endblock %}
 
-{% block site_css %}
+{% block page_css %}
   {% stylesheet 'firefox-dashboard' %}
```

Comment:
Just noticed this bundle loads `menu-resp.less`, which isn't needed on the two templates that inherit from this template (`firefox/interest-dashboard.html` and `firefox/tiles.html`).

Also, those two inheriting pages have their CSS in `site_css`, which results in a weird CSS load order.